modules/caddytls/automation.go

```diff
@@ -262,9 +265,9 @@ func (ap *AutomationPolicy) Provision(tlsApp *TLS) error {
 		// prevent issuance from Issuers (when Managers don't provide a certificate) if there's no
 		// permission module configured
 		noProtections := ap.isWildcardOrDefault() && !ap.onlyInternalIssuer() && (tlsApp.Automation == nil || tlsApp.Automation.OnDemand == nil || tlsApp.Automation.OnDemand.permission == nil)
-		failClosed := noProtections && hadExplicitManagers // don't allow on-demand issuance (other than implicit managers) if no managers have been explicitly configured
+		failClosed := noProtections && !ap.hadExplicitManagers // don't allow on-demand issuance (other than implicit managers) if no managers have been explicitly configured
```

Comment:
I made this change based on the inline comment, and it seems to make sense.  However, now looking at it again, this is a noop since it is the same check that results in the returned error below.  So either the inline comment here and/or I am a little confused, or we can just remove this `failClosed` var entirely.  Admittedly, this is my first time digging into this section of code, so I may just be misunderstanding the intent.

Reply:
I probably need more testing here :sweat_smile: